examples/migrationExample/src/main/java/io/realm/examples/realmmigrationexample/model/Migration.java

```diff
@@ -111,11 +108,9 @@ public long execute(Realm realm, long version) {
                 String type = petTable.getString(oldTypeIndex, i);
                 if (type.equals("dog")) {
                     petTable.setLong(typeIndex, i, 1);
-                }
-                else if (type.equals("cat")) {
+                } else if (type.equals("cat")) {
```

Comment:
This is against our C++ guideline, and how we have done most places. (although I personally prefer this new style). But I think the most important with the styling is to pick what's used by most Java devs.